web-app/helpers/certificate-designs.ts

```diff
@@ -28,29 +27,105 @@ function addText(canvas: Canvas, text: string, font: string, fillStyle: string,
   context.fillText(text, leftPos, rightPos);
 }
 
+// split long text into shorter lines
```

Comment:
Consider using JSDoc comment formatting here when stating what a function does. This will help IDE hints.

Reply:
Done